MPU/src/mpu.cpp

```diff
@@ -65,7 +63,7 @@ bool isCANLineOK()
 
 void setCANLineOK()
 {
-    canTest_wait.startTimer(1500);
+    canTest_wait.startTimer(2000);
```

Comment:
Why?

Reply:
BMS integration, we can decrease it if it works